config/Map.yaml

```diff
@@ -131,3 +131,20 @@ mappings:
     newAPI: "apiVersion: apiregistration.k8s.io/v1\nkind: APIServiceList\n"
     deprecatedInVersion: "v1.19"
     removedInVersion: "v1.22"
+  - deprecatedAPI: "apiVersion: storage.k8s.io/v1beta1\nkind: CSIDriver\n"
+    newAPI: "apiVersion: storage.k8s.io/v1\nkind: CSIDriver \n"
```

Comment:
might be an unintentional space before the newline

Reply:
Thanks for catching this, can confirm that it was unintentional, and the new commit removes it.